core/src/main/scala/io/finch/EndpointResult.scala

```diff
@@ -48,23 +52,22 @@ sealed abstract class EndpointResult[F[_], +A] {
     case _ => None
   }
 
-  def awaitOutput(d: Duration = Duration.Top)(implicit e: Effect[F]): Option[Try[Output[A]]] = this match {
+  def awaitOutput(d: ScalaDuration = Top)(implicit e: Effect[F]): Option[Try[Output[A]]] = this match {
```

Comment:
Perhaps, we don't have to alias `Duration` as `ScalaDuration` anymore?

Reply:
Makes sense as `twitter.Await` is replaced with `IO.unsafeRunTimed`. I fixed it.